configuration/configuration.go

```diff
@@ -501,4 +501,4 @@ var defaultKeycloakTesUser2Name = "testuser2"
 var defaultKeycloakTesUser2Secret = "testuser2"
 
 // Keycloak URL to be used in dev mode. Can be overridden by setting up keycloak.url
-var devModeKeycloakURL = "http://sso.demo.almighty.io"
+var devModeKeycloakURL = "http://sso.prod-preview.openshift.io"
```

Comment:
We could prefix it with default this variable or add a comment explaining that is a default one. It can be confusing to see this value hardcoded